lib/sycamore/sycamore/transforms/extract_entity.py

```diff
@@ -75,13 +76,15 @@ def __init__(
         self._prompt_formatter = prompt_formatter
 
     def extract_entity(self, document: Document) -> Document:
+        tt = TimeTrace("OaExtract")
```

Comment:
"entityExtract"?  I don't know what 0a is.

Reply:
Oa = OpenAI

Reply:
I'd recommend changing this to entityExtract since I can't find anything in there that forces this to be openai (the variable is even named self._llm.  If you're going to insist on leaving in something about openai, then either a) openAIEntityExtract or a comment to explain.  I'm going to have trouble remembering this even with the explanation.

Reply:
The class is called `OpenAIEntityExtractor`.  I think the label is fine; they should be short.